tests/test-backend-ops.cpp

```diff
@@ -1591,13 +1583,9 @@ struct test_mul_mat : public test_case {
         return 5e-4;
     }
 
-    size_t op_size(ggml_tensor * t) override {
-        size_t a = ggml_nbytes(t->src[0]) * n * nr[0] * nr[1];
-        size_t b = ggml_nbytes(t->src[1]) * m;
-        size_t c  = ggml_nbytes(t);
-        return a + b + c;
-
+    uint64_t op_flops(ggml_tensor * t) override {
         GGML_UNUSED(t);
+        return 2 * m * n * k * bs[0] * nr[0] * bs[1] * nr[1];
```

Comment:
Theoretically you would only need $m \cdot n \cdot k$ multiplications and $m \cdot \cdot (k - 1)$ additions. In practice however this is not going to make a difference due to $k >> 1$.

Reply:
I agree, but most sources seem use $2mnk$, so I thought that it would be more important to be consistent with the way other people measure FLOPS than to correct a small inaccuracy.